tutorials/foreign-object.html

```diff
@@ -0,0 +1,813 @@
+            you don't have to. It turns out the events that get triggered are quite inconsistent across all major browsers when interacting with 
+            <code>select</code>. As a result, the JointJS <code>paper</code> does not handle UI events related to the <code>select</code> 
+            element by default. This just means that <code>select</code>, for consistency reasons, is included by default in that 
+            <code>paper</code> <code>guard</code> option we've mentioned several times already. You as a user don't have to do anything about it,
+            but it's something to be aware of.
+        </p>
+
+        <p>
+            One last delightful quirk of working with <code>select</code> in Safari is that when <code>multiple</code> is set to <code>'true'</code>,
+            Safari does not allow you to select an <code>option</code> with a mouse, but does however allow selection via keyboard navigation.
+        </p>
+
+        <p>HTML Elements which show inconsistency in Safari are as follows:</p>
+
+        <ul>
+            <li><code>video</code></li>
+            <li><code>select</code></li>
+        </ul>
+        
+        <p>
+            As stated already, this isn't an exhaustive list of issues, so it's possible you may run into some other inconsistencies with Safari.
+            It's unfortunate that Safari still lags behind other modern browsers in relation to <code>foreignObject</code>, but there are also 
+            reasons to be positive. One reason to have a brighter outlook is that some of the major issues we experienced here are currently being 
+            worked on according to this <a href="https://bugs.webkit.org/show_bug.cgi?id=90738" target="_blank">bug</a> report, so hopefully in the 
+            not too distant future, the majority of these issues will cease to exist.
+        </p>
+
+        <h3>Conclusion</h3>
+
+        <p>
+            We are very happy that we can now confidently recommend the use of <code>foreignObject</code> to users of JointJS, and that it's
+            well-supported in all major browsers. We consider it a big step forward for creating diagrams with JointJS, and believe it will 
+            allow the creation of even more innovative and interactive diagramming solutions. If you want to start using HTML with JointJS, there
+            is no better time to do so, and we can't wait to see what you create.
+        </p>
+
+        <br>
+
+        <p> 
+            <a href="https://github.com/clientIO/joint" target="_blank">
+                <img src="./images/github.png" alt="github logo" width="16" style="vertical-align:text-bottom;margin:2px;">
+                <strong>Star JointJS on GitHub!</strong>
+            </a> 
+            If you are a fan of JointJS, this is an easy way to support our underrated project, and help boost our ranking on GitHub.
+        </p>
```

Comment:
Do we really want to include this? And if yes, does that mean that we should include it on all our tutorials? I'm not saying we shouldn't necessarily, but it would be inconsistent to just have it on this article.

Although it does feel out of place.